Expensify/App
# adding the link back

<If necessary, assign reviewers that know the area or changes well. Feel free to tag any additional reviewers you see fit.>

### Details
Adding back the link to UpWork.

### Fixed Issues
https://github.com/Expensify/Expensify/issues/158833

### Tests
On Web
1. Open expensi.cash on the login page
2. Verify that you see the copy "" and the link to upworks.

### QA Steps
Same as normal tests

### Tested On


### Screenshots


#### Web
![Screenshot 2021-04-23 at 13 43 54](https://user-images.githubusercontent.com/242568/115878617-a3eccf00-a440-11eb-8791-4ff577e6fbce.png)

#### Mobile Web
![Simulator Screen Shot - iPhone 11 - 2021-04-23 at 14 45 29](https://user-images.githubusercontent.com/242568/115880410-9afcfd00-a442-11eb-875d-4bce88e47a44.png)

#### Desktop
![Screenshot 2021-04-23 at 14 43 36](https://user-images.githubusercontent.com/242568/115880172-57a28e80-a442-11eb-8fa8-23650c74dc02.png)

#### iOS
![Simulator Screen Shot - iPhone 11 - 2021-04-23 at 14 32 20](https://user-images.githubusercontent.com/242568/115878698-bcf58000-a440-11eb-86b5-fefbac55bdc2.png)

#### Android
![Screenshot_1619185301](https://user-images.githubusercontent.com/242568/115880185-5b361580-a442-11eb-9efc-0e453c512a63.png)
